salt/states/elasticsearch.py

```diff
@@ -234,10 +234,10 @@ def index_template_absent(name):
 
 def index_template_present(name, definition, check_definition=False):
     '''
-    Ensure that the named index templat eis present.
+    Ensure that the named index template is present.
 
     name
-        Name of the index to add
```

Comment:
This is incorrect, it should be the index to add, not check, because if it doesn't exist, it gets added.

Reply:
Equally, if it does exist it might get changed instead of added.
I'll leave it as it was if you prefer.